ql/src/test/org/apache/hadoop/hive/ql/TestTxnCommands2.java

```diff
@@ -70,6 +70,10 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * TODO: this should be merged with TestTxnCommands once that is checked in
```

Comment:
Probably not.  These are long running test suites - combining them will make it worse and PTest is unable to parallelize anything within one suite

Reply:
Agreed.